README.md

```diff
@@ -509,12 +509,12 @@ Pyxel application file also can be converted to an executable or an HTML file wi
   Fill the area connected with the same color as (`x`, `y`) with color `col`.
 
 - `blt(x, y, img, u, v, w, h, [colkey])`<br>
-  Copy the region of size (`w`, `h`) from (`u`, `v`) of the image bank `img` (0-2) to (`x`, `y`). If negative value is set for `w` and/or `h`, it will reverse horizontally and/or vertically. If `colkey` is specified, treated as transparent color.
+  Copy the region of size (`w`, `h`) from (`u`, `v`) of the image bank `img` (0-2) to (`x`, `y`). If a negative value is set for `w` and/or `h`, it will reverse horizontally and/or vertically. If `colkey` is specified, that color will be treated as transparent. A resource file must first be loaded using the `load` function in order to copy from the image bank.
```

Comment:
It would also be helpful to mention that the first example, 01_hello_world.py, demonstrates the use of `Image.load`.